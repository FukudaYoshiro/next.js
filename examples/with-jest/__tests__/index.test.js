/* global it, expect */
import React from 'react'
import { shallow } from 'enzyme'
import App from '../pages/index.js'

it('App shows "Hello world!"', () => {
  const app = shallow(
    <App />
  )

  expect(app.find('p').text()).toEqual('Hello world!')
})
