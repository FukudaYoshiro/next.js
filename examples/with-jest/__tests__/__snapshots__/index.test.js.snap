exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div
  data-jsx={2648947580}>
  <p
    data-jsx={2648947580}>
    Hello World!
  </p>
</div>
`;
